pyproject.toml

```diff
@@ -239,9 +239,12 @@ extend-exclude = [
   "_typed_ops.pyi",
 ]
 
+[tool.ruff.format]
+docstring-code-format = true
+
 [tool.ruff.lint]
 # E402: module level import not at top of file
-# E501: line too long - let black worry about that
+# E501: line too long - let the formatter worry about that
```

Comment:
Maybe this is obsolete when using ruff as a formatter as well?

Reply:
It is not, ruff emits these warnings too